packages/boomerang/boomerang.1.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "boomerang"
+version: "1.1.0"
+maintainer: "root <root@solomon-X556UA>"
+authors: "root <root@solomon-X556UA>"
+homepage: ""
+bug-reports: ""
+license: ""
+dev-repo: ""
```

Comment:
These empty fields do need to be filled in; the GitHub repository locations should be sufficient.